documentation/tut/toy-example-from-scratch.rst

```diff
@@ -65,6 +65,8 @@ Install Flexmeasures and the database
             $ docker stop flexmeasures-tutorial-db
             $ docker stop flexmeasures-tutorial-fm
 
+        .. note:: A tip on Linux/MacOs ― You might have the ``docker`` command, but need `sudo` rights to execute it. ``alias docker='sudo docker'`` enables you to still run this tutorial.
```

Comment:
`macOS` seems to be the [preferred spelling](https://en.wikipedia.org/wiki/MacOS#macOS).